pandas/info.py

```diff
@@ -1,20 +0,0 @@
-"""
-pandas - a powerful data analysis and manipulation library for Python
-=====================================================================
-
-See http://pandas.pydata.org/ for full documentation. Otherwise, see the
```

Comment:
yeah should replace with *something*, maybe a pointer to the docs (plus some minmal stuff). I can simply define this directly I think (in __init__.py). Any thoughts on what this should say?